apps/emqx_authn/src/simple_authn/emqx_authn_redis.erl

```diff
@@ -138,7 +138,9 @@ authenticate(#{password := Password} = Credential,
                     {error, Reason}
             end;
         {error, Reason} ->
-            ?LOG(error, "['~s'] Query failed: ~p", [Unique, Reason]),
+            ?SLOG(error, #{msg => "query failed",
+                           unique => Unique,
```

Comment:
having a `_unique' field embedded  (which IMO should be refactored) is one thing.
exposing it in logs is another, if it's an ID, maybe log it as `authn_id` field in SLOG